sources/api/migration/migrations/v1.20.0/add-ntp-default-options-v0-1-0/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "add-ntp-default-options"
```

the package name in `Cargo.toml` needs to be the same as the directory name or the build won't be able to find the package:

```suggestion
name = "add-ntp-default-options-v0-1-0"
```